### PR TITLE
docs(start_planner): show typical usecase of start planner and limitaion

### PR DESCRIPTION
**Pull request by kyoichi-sugahara**
## Description

show typical usecase of start planner and limitaion

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_path_start_planner_module/README.md [hunk 1]
```diff
@@ -2,24 +2,73 @@
 
 ## Purpose / Role
 
-The Start Planner module is designed to generate a path from the current ego position to the driving lane, avoiding static obstacles and stopping in response to dynamic obstacles when a collision is detected.
+This module generates and plans a path for safely merging from the shoulder lane into the travel lane.
```
Comment (kosuke55) [inline planning/behavior_path_start_planner_module/README.md:5]: not just shoulder lane

File: planning/behavior_path_start_planner_module/README.md [hunk 2]
```diff
@@ -2,24 +2,73 @@
...
 
-Use cases include:
+Specifically, it includes the following features:
 
-- pull out from the side of the road lane to centerline.
+- The ability to automatically start from the shoulder lane.
+- When parked vehicles are present on the shoulder lane, the module generates a path that allows for starting with a gap of a specified margin.
+- If a collision with other traffic participants is detected while traveling on the generated path, it will stop as much as possible.
```
Comment (kosuke55) [inline planning/behavior_path_start_planner_module/README.md:11]: not just shoulder lane

**Comment by kosuke55** [comment]
coool images!

File: planning/behavior_path_start_planner_module/README.md (hunk 1 above)
Comment (kyoichi-sugahara) [reply planning/behavior_path_start_planner_module/README.md:5]: sorry fixed in https://github.com/autowarefoundation/autoware.universe/pull/6264/commits/29d143817702013686e86cc031616fdc219a83ff

File: planning/behavior_path_start_planner_module/README.md (hunk 2 above)
Comment (kyoichi-sugahara) [reply planning/behavior_path_start_planner_module/README.md]: sorry fixed in https://github.com/autowarefoundation/autoware.universe/pull/6264/commits/29d143817702013686e86cc031616fdc219a83ff

**Review by kosuke55 — APPROVED** [review]

**Pull request merged**